server/src/main/resources/config/unix/SeLionConfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "restartCycle": 60000,
     "uniqueSessionCount": 50,
+    "nodeRecycleThreadWaitTimeout": 300,
```

Comment:
@sebady I was unsure about this default. Do you think a 5min timeout is a reasonable default?